### ISSUE TITLE
“Diable Presence BroadCast” Problem (Suggestion)

### ISSUE THREAD

**Issue by codeneno** [RocketChat/Rocket.Chat#20147]
### Description:

When “Diable Presence BroadCast” We need a user interface to get user ative status（click user photo?）, “Diable Presence BroadCast” is very important to resolve many problem,but we can't get a user status immediately.

### Steps to reproduce:

1. 
2. 
3. 

### Expected behavior:
When “Diable Presence BroadCast” We need a user interface to get user ative status（click user photo?）, “Diable Presence BroadCast” is very important to resolve many problem,but we can't get a user status immediately.

### Actual behavior:

NO

### Server Setup Information:

- Version of Rocket.Chat Server: ALL
- Operating System: ALL
- Deployment Method: ALL
- Number of Running Instances: 22
- DB Replicaset Oplog: Enabled
- NodeJS Version: all
- MongoDB Version:all

### Client Setup Information

- Desktop App or Browser Version: all
- Operating System:all

**Comment by magicbelette** [comment]
I agree that Presence Broadcast is a bottleneck for large deployment. But I don't really get why is so different that sending message ? Should we also disable Notifications ?

**Comment by codeneno** [comment]
I think it's a bug.

**Comment by codeneno** [comment]
you can fllow me and sumit suggestion to RC

**Comment by codeneno** [comment]
is anybody here?

**Comment by codeneno** [comment]
hello

**Comment by codeneno** [comment]
hi
